src/constants/signTransactionPage.ts

```diff
@@ -0,0 +1,62 @@
+import { FeeBumpTransaction, Transaction } from "@stellar/stellar-sdk";
+
+export const FEE_BUMP_TX_FIELDS = (feeBumpTx: FeeBumpTransaction) => [
+  {
+    label: "Fee source account",
+    value: feeBumpTx.feeSource,
+  },
+  {
+    label: "Transaction Fee (stroops)",
+    value: feeBumpTx.fee,
+  },
+  {
+    label: "Number of existing signatures",
+    value: feeBumpTx.signatures.length,
```

Comment:
I think we might need to make these optional in case the provided XDR doesn't have the values. What do you think?

```suggestion
    value: feeBumpTx.signatures?.length,
```